webapp/components/wpt-amend-metadata.js

```diff
@@ -135,6 +146,21 @@ class AmendMetadata extends LoadingState(ProductInfo(PolymerElement)) {
     return link;
   }
 
+  hasHref(product) {
+    return product === 'chrome';
+  }
+
+  getSearchURLHref(testName) {
+    if (this.computePathIsATestFile(testName)) {
+      // Remove name flags and extensions: https://web-platform-tests.org/writing-tests/file-names.html
+      testName = testName.split('.')[0];
+    } else {
+      testName = testName.replace(/((\/\*)?$)/, '');
+    }
+
+    return 'https://bugs.chromium.org/p/chromium/issues/list?q=' + testName;
```

Comment:
This needs to be quoted. Compare the search results [with](https://bugs.chromium.org/p/chromium/issues/list?q=%22%2FFileAPI%2Ffile%2FFile-constructor%22&can=2) and [without](https://bugs.chromium.org/p/chromium/issues/list?q=%2FFileAPI%2Ffile%2FFile-constructor&can=2) quotes. And you can use a template string.

```suggestion
    return `https://bugs.chromium.org/p/chromium/issues/list?q="${testName}"`;
```

Reply:
Yup I agree